src/main/java/com/example/delivery_system/entity/Carriage.java

```diff
@@ -0,0 +1,39 @@
+package com.example.delivery_system.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Set;
+
+@Data
+@Entity
+@Table(name = "carriages")
+public class Carriage implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "capacity", nullable = false)
```

Comment:
You don't need to add a name property if it's the same as a field name. Also, camel case will be translated to snake case automatically (E.g.: nameOfField -> name_of_field)